### PR TITLE
Hw6 add posts page

### PR DESCRIPTION
**Pull request by KRotarrash**

### CODE REVIEW

File: src/components/molecules/Post/MainPost/MainPost.tsx [hunk 1]
```diff
@@ -0,0 +1,212 @@
...
+        </ContentLeft>
+      </ContentMain>
+      <ContentFooter size={size}>
+        <FlexContainer>
+          <LikeIconStyled></LikeIconStyled>
+          <LikeNumber>20</LikeNumber>
+          <DislikeIconStyled></DislikeIconStyled>
+        </FlexContainer>
+        <FlexContainer>
+          <FavoriteIconStyled></FavoriteIconStyled>
```
Comment (Arween) [inline src/components/molecules/Post/MainPost/MainPost.tsx:104]: <FavoriteIconStyled/>
<EllipsisIconStyled/>

File: src/components/molecules/Post/MainPost/MainPost.tsx [hunk 2]
```diff
@@ -0,0 +1,212 @@
...
+          <EllipsisIconStyled></EllipsisIconStyled>
```
Comment (Arween) [inline src/components/molecules/Post/MainPost/MainPost.tsx:105]:
```
<FavoriteIconStyled/>

<EllipsisIconStyled/>
```

File: src/components/pages/Posts/Posts.tsx [hunk 3]
```diff
@@ -36,27 +40,207 @@ export const PostsPage = () => {
       });
   }, []);
 
-  // console.log(posts);
+  const firstPosts = getPosts(0, 1);
+  const mediumPosts = getPosts(1, 4);
+  const smallPosts = getPosts(5, 6);
+
+  function getPosts(from: number, count: number) {
```
Comment (Arween) [inline src/components/pages/Posts/Posts.tsx]: `const getPosts = (from: number, count: number) => {...`

this function looks like https://developer.mozilla.org/ru/docs/Web/JavaScript/Reference/Global_Objects/Array/slice

File: src/components/molecules/Post/MainPost/MainPost.tsx [hunk 4]
```diff
@@ -0,0 +1,212 @@
+import styled from 'styled-components';
+import { Url } from 'url';
```
Comment (Arween) [inline src/components/molecules/Post/MainPost/MainPost.tsx:2]: import { Url } from 'url';  ??

File: src/components/molecules/Post/MainPost/MainPost.tsx [hunk 5]
```diff
@@ -0,0 +1,212 @@
...
+          <HeaderStyled href={'/posts/' + id} size={size}>
+            {title}
+          </HeaderStyled>
+        </ContentLeft>
+      </ContentMain>
+      <ContentFooter size={size}>
+        <FlexContainer>
+          <LikeIconStyled></LikeIconStyled>
+          <LikeNumber>20</LikeNumber>
+          <DislikeIconStyled></DislikeIconStyled>
```
Comment (Arween) [inline src/components/molecules/Post/MainPost/MainPost.tsx:101]: its looks like independent component

```
<LikeIconStyled></LikeIconStyled>
          <LikeNumber>20</LikeNumber>
          <DislikeIconStyled></DislikeIconStyled>
```

File: src/components/molecules/Post/MainPost/MainPost.tsx [hunk 6]
```diff
@@ -0,0 +1,212 @@
...
+const ContentRight = styled.div`
+  display: flex;
+  justify-content: end;
+  align-items: flex-start;
+  width: 256px;
+  margin-bottom: 20px;
+  display: flex;
+`;
+
+const HeaderStyled = styled.a<{ size: 'large' | 'medium' | 'small' }>`
```
Comment (Arween) [inline src/components/molecules/Post/MainPost/MainPost.tsx:186]: styled(Link)

use Link, not <a>

**Comment by KRotarrash** [comment]
complete

**Pull request closed without merging**